src/components/NewMovie/NewMovie.tsx

```diff
@@ -13,33 +52,42 @@ export const NewMovie = () => {
       <TextField
         name="title"
         label="Title"
-        value=""
-        onChange={() => {}}
+        value={title}
+        onChange={(value) => setTitle(value)}
         required
       />
 
       <TextField
         name="description"
         label="Description"
-        value=""
+        value={description}
+        onChange={(value) => setDescription(value)}
```

Comment:
```suggestion
        onChange={setDescription}
```
Here and across your code - you don't need to wrap your handler with additional function. Just directly pass your function to child component.